virttest/utils_libvirt/libvirt_network.py

```diff
@@ -78,6 +79,26 @@ def check_established(params):
     port_to_check = params.get("port_to_check", "4915")
     check_local = 'yes' == params.get("check_local_port", "no")
     ipv6_config = "yes" == params.get("ipv6_config", "no")
+    check_network_transport_conn_num = params.get("check_network_transport_conn_num")
+    check_socket_num = "yes" == params.get("check_socket_num", "no")
+    exp_num = params.get("expected_socket_num", "2")
+    sockets = None
+
+    def _check_socket():
+        _res = process.run(cmd, shell=True).stdout_text.split('\n')
+        global sockets
```

Comment:
I assume it may not be necessary to use global here to declare sockets as global variable unless you want to get sockets value out of check_established function scope